polling_stations/settings/base.py

```diff
@@ -185,6 +185,8 @@
 
 INSTALLED_APPS += PROJECT_APPS
 
+SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
+
```

Comment:
`base.py` already defines a session engine on line 109. If you're changing the session engine, remove that declaration for clarity